site/content/config.js

```diff
@@ -26,6 +26,7 @@ export const config = {
     { href: "/_all", name: "All" },
   ],
   github: "https://github.com/flowershow/flowershow",
+  discord: "https://discord.com/channels/1029670148824641576/1029670149470556172",
```

Comment:
I just used the link I got from the channel but please let me know if this should be different